### DISCUSSION TITLE
Trouble Create case in the Hive

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#4071] [Unsupported Versions]
I have three, airgapped, distributed SO grids running 2.3.40 or 2.3.50. In each, when clicking the "Create case in the Hive" button in kibana, I am receiving an error stating "Internal Server Error, The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application."  I don't believe these servers are overloaded.. The logs located in /opt/so/logs/thehive/ don't seem to help troubleshoot this issue. Screenshot below. Any thoughts?
![image](https://user-images.githubusercontent.com/12967094/116748771-e593f180-a9cd-11eb-8ae9-09ab963d530a.png)

**Comment by weslambert** [comment]
Is `so-soctopus` running?  Are there any clues in `/opt/so/log/soctopus.log`, or if you run docker logs `so-soctopus`?
